### DISCUSSION TITLE
Equivalent in Quarto of Rmarkdown `message = FALSE` for code chunks?

### DISCUSSION THREAD

**Discussion by lucygarner** [quarto-dev/quarto-cli#2782] [Q&A]
Hi,

Is there an equivalent in Quarto of Rmarkdown `message = FALSE` for code chunks? I don't seem to be able to suppress messages in the output html file?

Best wishes,
Lucy

**Comment by Pierre9344** [comment]
Hello,

The same option as in rmarkdown can be used. The next code chunk won't show any warning and message in the rendered document. 
````
```{r}
#| warning: false
#| message: false

summary(iris)
```
````

**Comment by lucygarner** [comment]
Thanks, can you set it globally using `knitr::opts_knit$set` as you can with Rmarkdown?

I tried to use `message: false` in the YAML header but it was ignored when rendering the .qmd.
```
execute: 
  echo: false
  message: false
```

**Comment by Pierre9344** [comment]
Normally it should work. I used the same syntax in the _quarto.yml file that does with a quarto website.
Maybe you can remove the "execute;" part for the yaml?

**Comment by Pierre9344** [comment]
I just tried it and it work without the execute:

````
---
title: "test"
format: html
editor: visual
warning: false
message: false
---

## Quarto

```{r}
warning("warning")
message("message")
cat("cat")
```
````

**Comment by lucygarner** [comment]
Thank you - I will give that a try. I was following the recommendation here: https://quarto.org/docs/computations/execution-options.html.
